deploy/dev-env.sh

```diff
@@ -0,0 +1,107 @@
+#!/bin/bash -e
+
+usage() {
+  cat <<EOF
+Usage: $0 [option]
+
+options:
+  -b, --rebuild              Build new image and push it to your local registry
+                               before cluster creation
+  -c, --recreate             Create a new cluster based on the last build image
+
+For debugging run "bash -x $0 [option]"
+EOF
+}
+
+exists(){
+  command -v "$1" >/dev/null 2>&1
+}
+
+step1_run_local_registry(){
+  registry_is_running=$(podman ps | grep "registry:2")
+  if [ -z "$registry_is_running" ]; then
+    sudo mkdir -p /var/lib/registry
+    ./run-registry.sh
+  fi
+}
+
+step2_build_container_image(){
+  ./build-container.sh --force --no-registry-tls --registry docker://127.0.0.1:5000
+}
+
+step3_push_to_local_registry(){
+  podman push \
+      --tls-verify=false \
+      localhost/opensuse/bubbles:master \
+      docker://127.0.0.1:5000/opensuse/bubbles:master
+}
+
+prepare_kcli(){
+  # Install kcli if not installed
+  if ! exists kcli; then
+      echo "kcli could not be found"
+      echo "(You need to have installed libvirt-dev or libvirt-devel in order for the installation to succeed)"
+      curl https://raw.githubusercontent.com/karmab/kcli/master/install.sh | bash
+  fi
+  # Copy id_rsa.pub into .kcli and add to /root/.ssh/authorized_keys if it does not exist
+  klcipub="$HOME/.kcli/id_rsa.pub"
+  if [ ! -f "$klcipub" ]; then
+    sshpub="$HOME/.ssh/id_rsa.pub"
+    cp $sshpub $klcipub
+    cat $sshpub | sudo tee -a /root/.ssh/authorized_keys > /dev/null
+  fi
+}
+
+step4_run_kcli(){
+  prepare_kcli
+  plan=bubbles
+  if [ -n "$(kcli list plan | grep $plan)" ]; then
+    kcli delete plan $plan --yes
+  fi
+  kcli create plan -f plan/cluster.yml -P ceph_dev_folder=$(readlink -v -f "../../../../../") -P registry=docker://127.0.0.1:5000 $plan
```

Comment:
Could you please explain which directory `ceph_dev_folder` should point to? Changing `../../../../../` to `../../../../` does not abort the script anymore, but i don't know if this path points to the right (or intended) directory.

Reply:
Now the script checks if the assumed path is used and informs if it is not.